kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -139,6 +142,24 @@
                         ++orderInGroup,
                         Width.LONG,
                         "S3 accelerated endpoint enabled");
+
+      CONFIG_DEF.define(S3_PROXY_HOST,
+              Type.STRING,
+              Importance.HIGH,
```

Comment:
I'd say these are probably medium or even low importance -- many (most?) users won't even use these settings, so high is probably overkill.